.eslintrc

```diff
@@ -19,16 +19,19 @@
     "jest": true
   },
   "rules": {
-    "max-len": [1, 80, 2],
+    "max-len": [1, 200, 2],
```

Comment:
Why did you change this rule? Please update it to `80`

Reply:
I have updated it